Assets/Scripts/DynamicCamera/CameraController.cs

```diff
@@ -0,0 +1,60 @@
+﻿namespace HarmonyQuest.DynamicCamera
+{
+    using UnityEngine;
+    using System.Collections.Generic;
+
+    public class CameraController : MonoBehaviour
+    {
+        [SerializeField]
+        private List<CameraBehavior> behaviors = new List<CameraBehavior>();
+
+        private Vector3 target;
+
+        private static CameraController inst;
+        public static CameraController instance
+        {
+            get
+            {
+                if (!inst)
+                {
+                    inst = GameObject.FindObjectOfType<CameraController>();
+                }
+                if (!inst)
+                {
+                    GameObject camera = Camera.main.gameObject;
+                    inst = camera.AddComponent<CameraController>();
+                }
+                return inst;
+            }
+        }
+
+        void Awake()
+        {
+            ToggleCamera<CameraFollowPlayer>();
```

Comment:
This is assuming that this CameraBehavior will always be there. It probably will be, but I'd avoid referencing any behaviors by name in this class unless absolutely necessary.

Reply:
I'll switch it to toggle the first CameraBehavior in the list if there is any, which should work better once I switch the list to be dynamic.